src/Microsoft.AspNet.Mvc.Core/Filters/AuthorizeFilter.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Security;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// An implementation of <see cref="IAsyncAuthorizationFilter"/> which corresponds to [Authorize]
+    /// </summary>
+    public class AuthorizeFilter : IAsyncAuthorizationFilter, IOrderedFilter
```

Comment:
Sorry, wasn't clear, need docs on all of the public/protected stuff. If it's an interface method you can just use `/// <inheritdoc />`